docs/spec/0.95.0/1_Overview.md

```diff
@@ -1,8 +1,28 @@
 # ElectionGuard Specification ![Version 0.95.0][green-badge-0.95.0]
 
-!!! info "Download Specification"
-    No web version of this specification is available at this time. [Download specification here][spec-0.95.0].
```

Comment:
Instead of removing this can you leave this link since the entire specification is being done in pieces. 

> Web version is under construction at this time. [Download full specification here]